Packs/Okta/ReleaseNotes/3_1_26.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Okta v2
+
+- %%UPDATE_RN%%
```

Comment:
Update the Release Notes with the appropriate message that reflects the changes that you made.
Use [this](https://xsoar.pan.dev/docs/documentation/release-notes#examples-and-best-practices) for reference.